problems/[230601] K Closest Points to Origin.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/k-closest-points-to-origin
+ *
+ * @param {number[][]} points
+ * @param {number} k
+ * @return {number[][]}
+ */
+const kClosest = function (points, k) {
+  return points
+    .sort((point1, point2) => {
```

Comment:
```suggestion
        .sort((point1, point2) => {
        const point1Distance = point1[0] ** 2 + point1[1] ** 2;
        const point2Distance = point2[0] ** 2 + point2[1] ** 2;

        return point1Distance - point2Distance;
      })
```
이번 문제에서는 distance라는 숫자만을 다루게 되니 a - b 형태의 sort를 사용해도 좋을 것 같아요.

예린님 잘 아시겠지만 참고용으로..
제곱 사용할 때는 `Math.pow(2, 2) = 4`의 형태로 사용할 수 있고 
제곱근의 형태는 `Math.sqrt(4) = 2`의 메서드를 사용할 수 있어요. 참고용으로 달아 놓습니다!

Reply:
추가 학습에 적혀있었군요😄 답변 달고 봤어요... 허허